thirdparty/CMakeLists.txt

```diff
@@ -30,17 +30,23 @@ ov_install_static_lib(xbyak ${OV_CPACK_COMP_CORE})
 #
 # zlib
 #
-
 if(ENABLE_SAMPLES OR ENABLE_TESTS)
```

Comment:
I suppose when we build for conan, we should disable samples. So, we don't have issues with zlib and nlohmann_json 
Secondly, when we build via conan, we need to set cmake options as `-DENABLE_SYSTEM_PUGIXML=ON -DENABLE_SYSTEM_PROTOBUF=ON -DENABLE_SYSTEM_TBB=ON`.

So, if we do it, we don't need to change this file, right?

Reply:
Good point :) I close the PR